frontend/apps/web/app/(mgmt)/[account]/jobs/[id]/layout.tsx

```diff
@@ -195,40 +195,14 @@ interface SidebarNav {
 function getSidebarNavItems(
   accountName: string,
   job?: Job,
-  isMetricsServiceEnabled?: boolean
+  isMetricsServiceEnabled?: boolean,
+  isJobHooksEnabled?: boolean
 ): SidebarNav[] {
   if (!job) {
     return [{ title: 'Overview', href: `` }];
   }
   const basePath = `/${accountName}/jobs/${job.id}`;
 
-  if (isDataGenJob(job) || isAiDataGenJob(job)) {
```

Comment:
this wasnt actually used and was overridden by the outside nav below this if clause